benchmarks/README.md

```diff
@@ -26,22 +49,34 @@ To run a specific dataset:
 For java:
 
 ```
-$ make java
+$ make java-benchmark
 $ ./java-benchmark $(specific generated dataset file name)
 ```
 
 For cpp:
 
 ```
-$ make cpp
+$ make cpp-benchmark
 $ ./cpp-benchmark $(specific generated dataset file name)
 ```
 
+## Benchmark datasets
+
+There's some big testing data not included in the directory initially, you need to 
+run the following command to download the testing data:
+
+```
+$ ./download_data.sh 
```

Comment:
Is this required to build the benchmarks? If so please move it to the head of this README.